src/client/components/Swipe/Swipe.jsx

```diff
@@ -77,7 +78,7 @@ class Swipe extends Component {
         });
       })
       .catch(function (error) {
-        alert(error.message);
+        console.log(error.message);
```

Comment:
just fyi, sashankh's PR #199 replaces this with toast, so this is gonna cause merge conflict

Reply:
I'll accept his changes when resolving this :D